src/hippo_notifier.rs

```diff
@@ -0,0 +1,14 @@
+pub struct ConnectionInfo {
+    pub url: String,
+    pub username: String,
+    pub password: String,
+}
+
+pub async fn register(bindle_id: &bindle::Id, conn_info: &ConnectionInfo) -> anyhow::Result<()> {
+    // TODO: username and password
```

Comment:
can this comment be removed now that this appears to be obtained from the ConnectionInfo?